www/front_src/src/Resources/ChecksIcon.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+
+import { useTranslation } from 'react-i18next';
+
+import SyncDisabledIcon from '@material-ui/icons/SyncDisabled';
+import SyncProblemIcon from '@material-ui/icons/SyncProblem';
+import { Tooltip } from '@material-ui/core';
+
+import { Resource } from './models';
+import {
+  labelChecksDisabled,
+  labelOnlyPassiveChecksEnabled,
+} from './translatedLabels';
+
+interface IconProps {
+  Component: (props) => JSX.Element;
+  title: string;
+}
+
+const Icon = ({ Component, title }: IconProps): JSX.Element => {
+  const { t } = useTranslation();
+
+  return (
+    <Tooltip title={t(title) as string}>
```

Comment:
The t function returns a string
```suggestion
    <Tooltip title={t(title)}>
```